Anlab.Core/Services/SlothService.cs

```diff
@@ -17,6 +17,8 @@ public interface ISlothService
     {
         Task<SlothResponseModel> MoveMoney(Order order);
         Task<bool> ProcessCreditCards(FinancialSettings financialSettings);
+
+        Task<bool> MoneyMovedCheck(FinancialSettings financialSettings);

```

Comment:
@jSylvestre These methods that return a bool would be good to have boolean sounding names, maybe like `MoneyHasMoved` or `VerifyMoneyHasMoved`, etc, for example.  Maybe for the processing method you don't need it, but this money check seems like a obvious one.